code/modules/admin/tabs/event_tab.dm

```diff
@@ -322,7 +322,7 @@
 	if(!admin_holder)
 		return
 
-	var/list/options = list("Weyland-Yutani", "High Command", "Provost", "Press", "CMB", "Other", "Cancel")
+	var/list/options = list("Weyland-Yutani", "High Command", "Provost", "Press", "CMB", "UPP", "TWE", "CLF", "Other", "Cancel")
 	var/answer = tgui_input_list(src, "Which kind of faxes would you like to see?", "Faxes", options)
```

Comment:
The entries in this list and the `switch(answer)` checks below need to match up, since `answer` is being assigned based on which of these options the user picks.
So they should either all be initialisms like they are now (in which case this line doesn't need changing), or they should be the full names like this:
```suggestion
	var/list/options = list(
		"Weyland-Yutani", "High Command", "Provost", "Press",
		"Colonial Marshal Bureau", "Union of Progressive Peoples",
		"Three World Empire", "Colonial Liberation Front",
		"Other", "Cancel")
	var/answer = tgui_input_list(src, "Which kind of faxes would you like to see?", "Faxes", options)
```